tqdm/_tqdm.py

```diff
@@ -267,8 +267,8 @@ def format_meter(n, total, elapsed, ncols=None, prefix='',
         format_sizeof = tqdm.format_sizeof
         rate_fmt = ((format_sizeof(inv_rate if inv_rate else rate)
                     if unit_scale else
-                    '{0:5.2f}'.format(inv_rate if inv_rate else rate))
-                    if rate else '?') \
+                    '{0:5.2f} '.format(inv_rate if inv_rate else rate))
+                    if rate else '? ') \
```

Comment:
I inserted spaces so the printed text would be more viewable (I don't know if I even used existing word in English since English is not my native language...I hope you understand)